Core/src/org/sleuthkit/autopsy/centralrepository/datamodel/AbstractSqlEamDb.java

```diff
@@ -1625,10 +1611,9 @@ public Long getCountArtifactInstancesKnownBad(CorrelationAttribute.Type aType, S
      * @throws EamDbException
      */
     @Override
-    public List<String> getListCasesHavingArtifactInstancesKnownBad(CorrelationAttribute.Type aType, String value) throws EamDbException {
-        if (aType == null) {
-            throw new EamDbException("Correlation type is null");
-        }
+    public List<String> getListCasesHavingArtifactInstancesKnownBad(CorrelationAttribute.Type aType, String value) throws EamDbException, CorrelationAttributeNormalizationException {
+        
+        String normalizeValue = CorrelationAttributeNormalizer.normalize(aType, value);
```

Comment:
Please change all the "normalizeValue" variables to "normalizedValue" for consistency